workers/social/lib/social/models/computeproviders/stacktemplate.coffee

```diff
@@ -337,6 +423,42 @@ module.exports = class JStackTemplate extends Module
 
     , (client, callback) ->
 
+      { group }    = client.r
+      { delegate } = client.connection
+      { stackTemplates } = group
+
+      stackTemplates = makeArray stackTemplates
+      id = @getId().toHexString()
+
+      if stackTemplates.length
+        unless id in stackTemplates
+          stackTemplates.push id
+      else stackTemplates = [id]
+
+      sharedStackTemplates = [id]
+
+      @setAccess client, 'group', (err) =>
+        group.modify client, { stackTemplates, sharedStackTemplates }, (err) =>
```

Comment:
why do we need that? and why we are keeping this list on `JGroup` instance?
we can get list of templates shared with group by simply `JStackTemplate.some {accessLevel: 'group'}`